packages/atlas/src/views/global/YppLandingView/sections/YppRewardSection.tsx

```diff
@@ -85,9 +85,10 @@ export const YppRewardSection: FC = () => {
         </CenteredLayoutGrid>
         <LayoutGrid>
           <TierCardWrapper colSpan={{ base: 12, sm: 10, md: 12, lg: 10 }} colStart={{ sm: 2, md: 1, lg: 2 }}>
-            {[...tiers].map((tier) => {
-              tier.rewards[tier.rewards.length - 1] = getTierRewards('diamond')?.referral || 0
-              return <TierCard key={tier.tier} {...tier} />
+            {tiers.map((tier) => {
+              const modifiedRewards = [...tier.rewards.slice(0, -1), getTierRewards('diamond')?.referral || 0]
+              const modifiedTier = { ...tier, rewards: modifiedRewards }
+              return <TierCard key={tier.tier} {...modifiedTier} />
```

Comment:
That works just due to how react overwrite props you could save one spread of `tier`:
```suggestion
              return <TierCard key={tier.tier} {...tier} rewards={modifiedRewards} />
```